flocker/node/agents/gce.py

```diff
@@ -0,0 +1,385 @@
+# -*- test-case-name: flocker.node.agents.functional.test_gce -*-
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+A GCE Persistent Disk (PD) implementation of the ``IBlockDeviceAPI``.
+
+The following resources are helpful to referrence while maintaining this
+driver:
+- Rest API: https://cloud.google.com/compute/docs/reference/latest/
+- Python Client: https://cloud.google.com/compute/docs/tutorials/python-guide
+- Python API: https://google-api-client-libraries.appspot.com/documentation/compute/v1/python/latest/
+- Python Oauth: https://developers.google.com/identity/protocols/OAuth2ServiceAccount#authorizingrequests
```

Comment:
I'm hoping adding a `# noqa` to the end of the line should do it.  If that doesn't work, I'll look into something more exciting.